run.sh

```diff
@@ -7,4 +7,4 @@ migrate)
     ;;
 esac
 
-bundle exec unicorn -p $UNICORN_PORT
+exec bundle exec unicorn -p $UNICORN_PORT
```

Comment:
This replaces the pid one with the unicorn process rather than having a sh process with a single child of bundle/unicorn